### PR TITLE
Trivial punctuation and typo edit.

### PR DESCRIPTION
**Pull request by christianromney**
Very much looking forward to your talk at the Conj!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,10 +62,10 @@ that the rule will create. It is comprised of a single predicate.
 Here is a rule that will infer an `uncle` relationship from existing data:
 
 ```
-  uncle(Nibling,Uncle) :- parent(Nibling,Parent), brother(Parent,Uncle).
+  uncle(Sibling,Uncle) :- parent(Sibling,Parent), brother(Parent,Uncle).
```
Comment (quoll) [inline README.md:65]: "Nibling" is the correct term here. It is a gender-neutral form for niece or nephew.

**Comment by christianromney** [comment]
Oh, I didn’t know that—thank you for teaching me something!

**Comment by CampingScorpion** [comment]
It makes me hungry, though :taco:

**Comment by quoll** [comment]
@christianromney Do you want to put the PR back in? The other changes were fine. I was thinking you might just update the PR, but I should have merged it and then rolled back to "Nibling".

I appreciate any PRs, and yours was welcome :)

**Pull request closed without merging**